### ISSUE TITLE
'Base' object has no attribute 'rpm_path'

### ISSUE THREAD

**Issue by cibo94** [rh-lab-q/rpg#143]
Output of rpg log:

```
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/rpg/gui/wizard.py", line 636, in initializePage
    self.buildLocationEdit.setText(str(self.base.rpm_path))
AttributeError: 'Base' object has no attribute 'rpm_path'
```

**Comment by cibo94** [comment]
and srpm too:
```
Traceback (most recent call last):
  File "/home/onionka/Projects/rpg/rpg/gui/wizard.py", line 677, in validatePage
    self.base.build_packages("fedora-21-x86_64")
  File "/home/onionka/Projects/rpg/rpg/__init__.py", line 172, in build_packages
    distro, arch)
  File "/home/onionka/Projects/rpg/rpg/package_builder.py", line 61, in build
    self.build_rpm(srpm_path, distro, arch)
NameError: name 'srpm_path' is not defined
```

**Comment by regeciovad** [comment]
Did you merge master branch? Because I am sure srpm_path was added

**Comment by jsilhan** [comment]
It was changed in Bse from `rpm_path` to `srpm_path`. We finally don;t build local RPM, just SRPM. @regeciovad can you fix it in GUI, please?

**Comment by regeciovad** [comment]
I was trying to fix it, but it is more complex problem.  Rpm_path is now remove and I add base_dir instead. Srpm_path can't be here - srpm is created after first error. What about build_packages? Should I delete it and use only base.build_srpm ?

**Comment by jsilhan** [comment]
Ok, we'll look at it at the meeting. Now continue with #117 .

**Comment by jsilhan** [comment]
I believe this is fixed by f968c65
